kernel/include/kernel_internal.h

```diff
@@ -227,10 +227,10 @@ void z_mem_manage_init(void);
  */
 void z_mem_manage_boot_finish(void);
 
-#define LOCKED(lck) for (k_spinlock_key_t __i = {},			\
```

Comment:
This should also be moved out of this PR

Reply:
Duplicate.